src/cli/cli.cc

```diff
@@ -118,28 +118,41 @@ StatusCode CompleteInstall(AkliteClient &client) {
     const auto current{client.GetCurrent()};  // returns Target a device is booted on
     if (current.Sha256Hash() != pending.Sha256Hash()) {
       // ostree rollback, aka the bootloader driven rollback
+      LOG_INFO << "Installation has failed, a device was rolled back to " << current.Name();
```

Comment:
I don't think "a device" makes sense in this context - its either just "device" or "the device".

Reply:
fixed.

Reply:
@doanac I think it's ready to be merged.